app/config/config.yml

```diff
@@ -36,7 +36,7 @@ framework:
     csrf_protection: ~
     validation:      { enable_annotations: true }
     templating:
-        engines: ['twig']
+        engines: ['twig','php']
```

Comment:
Ok ok lets talk about it, I just following the recommendations made in this [symfony commit](https://github.com/symfony/symfony/commit/1dbf67ed667c99f0d08ad61bf2bf4d2ec9fbada1) because the `templating.helper.router` is needed by `knplabs/knp-paginator-bundle` prior to 2.4.2